src/pages/home/report/ReportActionsList.js

```diff
@@ -136,12 +136,22 @@ function ReportActionsList({
     const route = useRoute();
     const opacity = useSharedValue(0);
     const userActiveSince = useRef(null);
-    const [currentUnreadMarker, setCurrentUnreadMarker] = useState(null);
+    const prevReportID = useRef(null);
+    const unreadActionSubscription = useRef(null);
+    const markerInit = () => {
+        if (!cacheUnreadMarkers.has(report.reportID)) {
+            return null;
+        }
+        return cacheUnreadMarkers.get(report.reportID);
+    };
+    const [currentUnreadMarker, setCurrentUnreadMarker] = useState(markerInit);
     const scrollingVerticalOffset = useRef(0);
     const readActionSkipped = useRef(false);
     const hasHeaderRendered = useRef(false);
     const hasFooterRendered = useRef(false);
     const reportActionSize = useRef(sortedReportActions.length);
+    const lastReadRef = useRef(report.lastReadTime);
```

Comment:
Why we need to store this in ref. Can we access report.lastReadTime directly?

Reply:
See Line 208